components/table/__tests__/index.test.tsx

```diff
@@ -117,6 +117,29 @@ describe('Table', () => {
     expect(() => wrapper.unmount()).not.toThrow()
   })
 
+  it('should be possible to update the row', () => {
+    const operation = (actions: cellActions) => {
+      return (
+        <button
+          onClick={() => actions.update({ property: 'test', description: 'test', operation })}>
```

Comment:
`actions.update` is it possible to fill in only part of the parameters: (we help users merge objects automatically)
```jsx
actions.update({ description: 'test' })
```

Reply:
in theory we have access to the previous object when i map so i can spread what the user give us yeah.

Reply:
Yes, most of the time, users only need to change part of the data, it's very friendly if we don't need to pass all the values in the parameter. (we can merge the object with the last one.)